pkg/master/master.go

```diff
@@ -513,7 +513,12 @@ func (m *Master) getServersToValidate(c *Config) map[string]apiserver.Server {
 			port = 4001
 		}
 		// TODO: etcd health checking should be abstracted in the storage tier
-		serversToValidate[fmt.Sprintf("etcd-%d", ix)] = apiserver.Server{Addr: addr, Port: port, Path: "/health", Validate: etcdutil.EtcdHealthCheck}
+		serversToValidate[fmt.Sprintf("etcd-%d", ix)] = apiserver.Server{
+			Addr:        addr,
+			EnableHTTPS: etcdUrl.Scheme == "https",
+			Port:        port,
+			Path:        "/health",
+			Validate:    etcdutil.EtcdHealthCheck}
```

Comment:
A tiny nit, but these look better if you end with a new line, e.g.

```
  blah: blah,
}
```

I'll LGTM anyway, since it is done this way elsewhere. Please update if you see this before the merge queue merges, though :) :)